docs/porting/libraries.md

```diff
@@ -1,5 +1,153 @@
-# 🔧 Porting your libraries
+# Porting Libraries to .NET Core
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+blah blah some intro
+
+## Prerequisites
+
+Before you begin, understand that this article assumes that you are using Windows and Visual Studio.
+
+It's recommended that you read the [Porting Overview](overview.md) article before you begin.  It will give you an idea of the journey you're about to embark on.
+
+## Getting your dependencies and .NET Framework APIs in check
+
+The two biggest difficulties in porting libraries to .NET Core is resolving issues with 3rd party dependencies and .NET Framework APIs that aren't compatible with .NET Core.
+
+If you haven't resolved issues with 3rd party dependencies, [please do so before continuing](overview.md).
+
+Next, make sure you [understand the API Portability tool](tooling.md) and can generate portability reports for targeting .NET Core.  You'll have to spend some time analyzing your assemblies and coming up with a plan for what to do with any incompatible types used.
+
+Here's a set of guidelines and questions you may find helpful.
+
+1. Understand where in the code each incompatible type is being used.
+
+   a.  Understand the nature of the incompatible types.  Are they small in number, but used frequently?  Are they large in number, but used infrequently?  Is their use concentrated, or is it spread throughout your code?
+   
+   b. Is it easy to isolate code that's largely incompatible from code that's large compatible?
+   
+   c. Would you need to refactor your code so that you can deal with incompatible types better?
+
```

Comment:
The questions are good ones to ask, I but it's a bit unclear how the answers would change what the reader would do. Maybe provide a few examples to illustrate alternative strategies?